### ISSUE TITLE
move all strings to i18n

### ISSUE THREAD

**Issue by louneskmt** [DecouvreBitcoin/sovereign-university#30]
Follow-up of #19. 

Move all hard-coded strings to `public/locales/en.json` and use i18n in the code.

```tsx
import { useTranslation } from 'react-i18next';

export const Component = () => {
    const { t } = useTranslation();

    // Access the strings by key:
    t('words.resources')
    t('components.resource-rate.public-grade')
}
```

See https://www.i18next.com/ and https://react.i18next.com/ for more details.

With VS Code, you can install the useful `i18n ally` extension. It will display in place the actual string instead of the function call with key, and there is a nice tool to edit strings:

![Image](https://github.com/DecouvreBitcoin/sovereign-university/assets/38882571/81111daf-3296-456d-b524-0211d4f3dc3d)

![Image](https://github.com/DecouvreBitcoin/sovereign-university/assets/38882571/fb94c1e0-6c60-4a35-bb63-39e244bac6b4)

![Image](https://github.com/DecouvreBitcoin/sovereign-university/assets/38882571/18e7858d-2467-4070-a275-94b9c5851bb5)

**Comment by trigger67** [comment]
Fixed in 2c9f82f and 8a37b47

**Comment by louneskmt** [comment]
Thanks @trigger67!
